Formula/btop.rb

```diff
@@ -0,0 +1,51 @@
+class Btop < Formula
+  desc "Resource monitor. C++ version and continuation of bashtop and bpytop"
+  homepage "https://github.com/aristocratos/btop"
+  url "https://github.com/aristocratos/btop/archive/refs/tags/v1.1.2.tar.gz"
+  sha256 "997376596cc6ca808b5a26f9fe25da274c19c2970aaf757943a009f4c28b758c"
+  license "Apache-2.0"
+
+  depends_on "coreutils" => :build
+
+  on_macos do
+    depends_on "gcc" => [:build, :test]
+  end
+
+  fails_with :clang # -ftree-loop-vectorize -flto=12 -s
+  # GCC 10 at least is required
+  fails_with gcc: "5"
+  fails_with gcc: "6"
+  fails_with gcc: "7"
+  fails_with gcc: "8"
+  fails_with gcc: "9"
+
+  def install
+    system "make", "CXX=#{ENV.cxx}", "STATIC=true", "STRIP=true"
```

Comment:
Are we building static libraries here? Because generally we don't want that.

Reply:
I think this is linking statically with `libstdc++`